.travis.yml

```diff
@@ -10,6 +10,7 @@ service:
 
 env:
   - GO111MODULE=on
```

Comment:
There should be only one line:

```
- GO111MODULE=on GOPROXY=https://proxy.golang.org
```

https://docs.travis-ci.com/user/environment-variables/#defining-multiple-variables-per-item